### PR TITLE
Make the OSTree module use strict Content-Security-Policy

### PR DESCRIPTION
**Pull request by stefwalter**

### CODE REVIEW

File: pkg/ostree/index.html [hunk 1]
```diff
@@ -206,21 +211,6 @@ <h3 ng-if="os_list.length > 1">{{ os }}</h3>
...
 </script>
 
-    <script>
-        require([
-            "base1/cockpit",
-            "translated!base1/po",
-            "base1/angular",
-            "updates/app"
-        ], function (cockpit, po, angular) {
-            cockpit.locale(po);
```
Comment (petervo) [inline pkg/ostree/index.html]: Should the locale and translate calls be moved to app.js?

File: pkg/ostree/index.html [hunk 2]
```diff
@@ -26,8 +26,13 @@
   <link href="ostree.css" rel="stylesheet">
   <script src="../base1/bundle.js"></script>
   <script src="ostree.js"></script>
+
+  <!-- debug:start -->
+  <script src="app.js"></script>
```
Comment (petervo) [inline pkg/ostree/index.html]: Should we load client.js here too. To make sure the module is fully loaded by the time we get to ng-app.

File: pkg/ostree/index.html (hunk 1 above)
Comment (stefwalter) [inline pkg/ostree/index.html]: Moved.

File: pkg/ostree/index.html (hunk 2 above)
Comment (stefwalter) [inline pkg/ostree/index.html]: It doesn't load until it's bundled, as it's an anonymous AMD module.

Comment (petervo) [inline pkg/ostree/index.html]: Did the see the angular error though in one of the tests? I was figuring it was a loading race of some kind but maybe it was something else

Comment (petervo) [inline pkg/ostree/index.html]: Happened here again with fedora atomic.

**Comment by stefwalter** [comment]
Hopefully this fixes it.

**Pull request closed without merging**